core/src/avm2/verify.rs

```diff
@@ -585,25 +588,31 @@ pub fn verify_method<'gc>(
             | AbcOp::Jump { offset } => {
                 let adjusted_result = adjust_jump_to_idx(i, *offset, true)?;
                 *offset = adjusted_result.1;
-                if let Some(jump_sources) = potential_jump_targets.get_mut(&adjusted_result.0) {
-                    if let JumpSources::Known(sources) = jump_sources {
-                        sources.push(i);
-                    }
+                if let Some(sources) = potential_jump_targets.get_mut(&adjusted_result.0) {
```

Comment:
Could these all use the Entry api instead of manual `if` and `insert`?
As in, smth like
```rs
potential_jump_targets.entry(&adjusted_result.0)
    .or_default().push(JumpSource::JumpFrom(i));
```